Flick/Cells/MediaSearchResultTableViewCell.swift

```diff
@@ -67,6 +66,14 @@ class MediaSearchResultTableViewCell: UITableViewCell {
         }
     }
 
+    func configure(media: Media) {
+        nameLabel.text = media.title
+        if let pictureUrl = URL(string: media.posterPic ?? ""), let pictureData = try? Data(contentsOf: pictureUrl) {
+            let pictureObject = UIImage(data: pictureData)
+            posterImageView.image = pictureObject
+        }
```

Comment:
would we still have to do this with KingFisher? if so, I could make it an extension

Reply:
Oh I was just gonna say that we don't have to do this anymore with kingfisher (I don't think)